webapp/packages/core-connections/src/DBDriverResource.ts

```diff
@@ -27,22 +28,25 @@ export const NEW_DRIVER_SYMBOL = Symbol('new-driver');
 export type NewDBDriver = DBDriver & { [NEW_DRIVER_SYMBOL]: boolean; timestamp: number };
 export type DBDriverResourceIncludes = Omit<DriverListQueryVariables, 'driverId'>;
 
-@injectable(() => [ServerConfigResource, GraphQLService, WorkspaceConfigEventHandler, AppAuthService])
+export type DriverPropertyInfo = DriverPropertyInfoFragment;
+
+@injectable(() => [SessionDataResource, GraphQLService, WorkspaceConfigEventHandler, AppAuthService])
 export class DBDriverResource extends CachedMapResource<string, DBDriver, DBDriverResourceIncludes> {
   get enabledDrivers() {
     return this.values.filter(driver => driver.enabled).sort(this.compare);
   }
 
   constructor(
-    private readonly serverConfigResource: ServerConfigResource,
+    private readonly sessionDataResource: SessionDataResource,
```

Comment:
Is it true that driver objects may change depending on the session?

Reply:
If the only `DBDriverExpertSettingsResource` will change, it's better to target`sync` it with `SessionDataResource` together with sync with `DBDriverResource`